### DISCUSSION TITLE
macOS: cp: cannot stat 'mount/ddev_config/[…]/': Operation not permitted

### DISCUSSION THREAD

**Discussion by slowcoder76** [ddev/ddev#5368] [General]
Sorry if I missed the solution somewhere, but my English is not very good. DDEV worked fine all the time and now I've been sitting for 2 days and don't know how to get it to work. Under logs -s web always comes:

```
cp: cannot stat '/mnt/ddev_config/nginx_full/._nginx-site.conf': Operation not permitted
cp: cannot stat '/mnt/ddev_config/nginx_full/._seconddocroot.conf.example': Operation not permitted
cp: cannot stat '/mnt/ddev_config/nginx_full/._README.nginx_full.txt': Operation not permitted
```
How do I get it to work again?

Regards

**Comment by rfay** [comment]
Hi @slowcoder76 - If you'll provide the output of `ddev debug test` either as an attached file or as a gist on gist.github.com I may be able to help. 

Mostly: Your docker provider isn't working for you. So restart it, reset it, or switch to another one like Orbstack.

Also, if you're using Colima it's possible that your project is not in the home directory (where it must be by default)
